Maven.gitignore

```diff
@@ -7,3 +7,6 @@ release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
+
+# Exclude maven wrapper
+!/.mvn/wrapper/maven-wrapper.jar
```

Comment:
The ``maven-wrapper.jar`` is excluded through the catch-all rule ``.jar``. However, the usecase of the maven-wrapper is, to avoid the need of having maven installed on a machine, where I want to build a just-checked-out project. Therefore, the ``maven-wrapper.jar`` needs to be excluded from the the ``.gitignore`` in order to be able to check it in along with the source code.